pages/accountLists/[accountListId]/contacts/Contacts.graphql

```diff
@@ -26,6 +24,12 @@ query Contacts(
   }
 }
 
+fragment contactFragment on Contact {
```

Comment:
I don't think that `ContactRow` is used in any other queries. What do you think about just adding `id` and `avatar` to the existing `ContactRow` fragment instead of creating a new one?